python/lsst/pipe/tasks/mocks/mockCoadd.py

```diff
@@ -197,8 +197,8 @@ def buildInputImages(self, butler, obsCatalog=None, truthCatalog=None, tract=0):
                 if status:
                     simSrcRecord = simSrcCatalog.addNew()
                     simSrcRecord.setCoord(truthRecord.getCoord())
-                    simSrcRecord.setL(self.objectIdKey, truthRecord.getId())
-                    simSrcRecord.setL(self.exposureIdKey, obsRecord.getId())
+                    simSrcRecord.setI(self.objectIdKey, truthRecord.getId())
+                    simSrcRecord.setI(self.exposureIdKey, obsRecord.getId())
```

Comment:
There is much confusion over `L` vs `I` in the C++ interface layer. Python (2 or 3) both use at least 64-bits for normal `int` type but in most cases C++ treats `I` as a 32-bit integer and that would indeed be too short for our 64-bit IDs.

See the discussion on [RFC-227](https://jira.lsstcorp.org/browse/RFC-227).